<launch>

  <arg name="data_dir"/>
  
  <group ns="deliver_server">
    <rosparam command="load" file="$(find bmce_apps)/cfg/deliver_params.yaml" />
  </group>
 
  <!-- main delivery launch -->
  <include file="$(find pr2_delivery)/launch/deliver_server.launch" />

  <node pkg="map_store" type="map_manager" name="map_manager" output="screen"/>
  
  <node name="delivery_exec" pkg="pr2_delivery" type="call_deliver.py">
    <param name="data_dir" value="$(arg data_dir)" />
  </node>
  
</launch>
